tools/vf-component-library/gulpfile.js

```diff
@@ -42,3 +47,33 @@ gulp.task('dev', gulp.series(
   'vf-build-search-index',
   gulp.parallel('watch','vf-watch')
 ));
+
```

Comment:
Would be best if this was a separate gulp file (to keep separation of concerns). Happy to do that if you'd like.

Reply:
that line? 

Happy for you to fix the gulp tasks I wrote into one `series` one and put it in the file system you've got for all the tasks.

Reply:
Just added a commit that does a tidy and fixes a couple missing tags